src/Message/HostedCheckoutDecryptReturnUrlRequest.php

```diff
@@ -51,7 +50,7 @@
  *   }
  * </code>
  */
-class HostedCheckoutDecryptReturnUrlRequest extends ExtendedAbstractRequest
+class HostedCheckoutDecryptReturnUrlRequest extends AbstractRequest
```

Comment:
Decryption of URL is not actually an extended API as shown in the docs https://developers.bluesnap.com/v8976-Tools/docs/encrypt-parameters